doc/api.rst

```diff
@@ -0,0 +1,236 @@
+.. _api_ref:
```

Comment:
This file looks like a copy-paste of the pyRiemann API.

IMO, pyRiemann-qiskit uses pyRiemann as dependency, and contains only quantic methods (QuanticSVM, QuanticVQC, etc).
Consequently, its API must make reference only to these new quantic methods.